modeller/chapter.md

```diff
@@ -247,16 +254,18 @@ class Scene(object):
 
 ### Nodes
 In the Scene's `render` function, we call `render` on each of the items in the Scene's `node_list`. But what are the elements
-of that list? We call them Nodes. 
-Conceptually, a `Node` is anything that can be placed in the scene.
-In object oriented software, we write `Node` as an abstract base class. Any classes that represent objects to be placed in the `Scene` will inherit from `Node`. 
+of that list? We call them *nodes*. 
+Conceptually, a node is anything that can be placed in the scene.
+In object-oriented software, we write `Node` as an abstract base class. Any classes that represent objects to be placed in the `Scene` will inherit from `Node`. 
```

Comment:
Similarly, removed capital from "node" when it's a generic node, and put it in code format when referring to objects of  class `Node`. I italicized the term on first use.